jx_sqlite/aggs_table.py

```diff
@@ -487,19 +490,20 @@ def _groupby_op(self, query, frum):
         groupby = []
         for i, e in enumerate(query.groupby):
             column_number = len(selects)
-            sql_type, sql = e.value.to_sql(schema)[0].sql.items()[0]
-            groupby.append(sql)
-            selects.append(sql + " AS " + e.name)
-
-            index_to_column[column_number] = ColumnMapping(
-                is_edge=True,
-                push_name=e.name,
-                push_column=column_number,
-                push_child=".",
-                pull=get_column(column_number),
-                sql=sql,
-                type=sql_type_to_json_type[sql_type]
-            )
+            for s in e.value.to_sql(schema):
+                sql_type, sql = s.sql.items()[0]
```

Comment:
`to_sql()[0]` is a relatively safe pattern to follow until we encounter properties that return more than one column (like any select * example).  `.sql.items()[0]` is not a safe pattern:  You must iterate through all the datatypes OR simply pull the type you are expecting:  `s.sql.b`, `s.sql.s`.  In your case you must loop.